client/deis.py

```diff
@@ -1271,6 +1266,61 @@ def domains_list(self, args):
         else:
             raise ResponseError(response)
 
+    def git(self, args):
+        """
+        Valid commands for git:
+
+        git:remote          Adds git remote of application to repository
+
+        Use `deis help [command]` to learn more.
+        """
+        return
```

Comment:
I've just updated that now to exit with a message suggesting viewing the help for the git command.